lib/interceptor.js

```diff
@@ -318,10 +318,10 @@ Interceptor.prototype.match = function match(options, body, hostNameOnly) {
 
     // special logger for query()
     if (queryIndex !== -1) {
-        this.scope.logger('matching ' + matchKey + '?' + queryString + ' to ' + this._key +
+        this.scope.logger('matching ' + matchKey + ( path || "" )+ '?' + queryString + ' to ' + this._key +
```

Comment:
I think you'll want to use pathname instead of path for instances where query params are present.